kernel/src/cpuid.c

```diff
@@ -0,0 +1,38 @@
+#include <stdint.h>
+
+#include "io.h"
+#include "cpuid.h"
+
+void print_reg(uint32_t r) {
```

Comment:
Add static since its not used anywhere else.
```suggestion
static void print_reg(uint32_t r) {
```